### ISSUE TITLE
Test 'httpsCallable' using 'initializeTestApp'

### ISSUE THREAD

**Issue by jgavazzisp** [firebase/firebase-tools#3416]
### [REQUIRED] Environment info

**firebase-tools:** 9.12.0
**Platform:** Mac

### [REQUIRED] Test case

I'm trying to test a 'httpsCallable' using the 'initializeTestApp' from "@firebase/testing".
The test fails with an "internal" error and no more description as shown below:
![image](https://user-images.githubusercontent.com/309389/119777722-adbf9180-bf1a-11eb-8b83-69a92d7e53ac.png)

### [REQUIRED] Steps to reproduce

To reproduce it, all it is needed is a 'httpsCalled' function such as:

index.js
```
exports.acceptInvite = functions.https.onCall((data, context) => {
    console.log('functions called')
});
```
and a test such:

functions.spec.js
```
it("test invite is created", async () => {
        const functions = firebase.initializeTestApp({
            projectId: projectId,
            auth: {uid: "alice", email: "alice@example.com"}
        }).functions()

        functions.useFunctionsEmulator("localhost:5001")

        const acceptInvite = functions.httpsCallable('acceptInvite')
        await acceptInvite({email: 'email_to_be_invited@gmail.com'})
    })
```

### [REQUIRED] Expected behavior

The httpsCallable function to be called with the token setup in the 'initializeTestApp'

### [REQUIRED] Actual behavior

Crash with 'internal' error.

Thanks

**Comment by samtstern** [comment]
@jgavazzisp a few things here:

- We moved `@firebase/testing` to `@firebase/rules-unit-testing`, so the package you're using is out of date.
- The utilities in `rules-unit-testing` such as `initializeTestApp` are only meant for testing Security Rules interactions with Realtime Database / Cloud Firestore / Cloud Storage. To test callable functions you'll want to use something like `firebase-functions-test`.

You can see some testing examples here:
https://github.com/firebase/quickstart-testing/

Since this is not a CLI bug I am going to close this issue but if you have more testing questions feel free to file an issue on the quickstart-testing repo!

**Comment by jgavazzisp** [comment]
Hi Sam, thanks for the clarification, update my test code.

**Comment by jgavazzisp** [comment]
@samtstern quick question, the tests run fine when I set a script and run them using firebase:exec. I was just wondering if I could run individual tests using webstorm where I start my emulators separately. I've tried to set up the environment variables to point to the emulators but it seems Firestore always try to hit prod instead.

![image](https://user-images.githubusercontent.com/309389/120032156-e01ddb80-c04d-11eb-8141-1ef8a3ceca13.png)

Those are the env variables. 

For simplicity, I tried to run the individual tests using the firebase quickstart-testing > unit-test-cloud-functions.

Thanks
